File: Data Structures & Algorithms/two-integer-sum-ii/submission-3.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int n = numbers.length;
        for(int i = 1;i<n;i++){
            for(int j = i;j<n;j++){
                if(numbers[j] + numbers[j-i] == target){
                    return new int[]{j-i+1, 1+j};
                }
            }
        }
        return new int[0];
    }
}

File: Data Structures & Algorithms/find-target-in-rotated-sorted-array/submission-3.py
class Solution:
    def search(self, nums: List[int], target: int) -> int:
        n = len(nums)
        if n==1 and nums[0] == target:
            return 0
        left,right=0,n-1

        while left<right:
            mid = left + (right-left)//2
            if nums[mid]==target:
                return mid
            elif nums[left]>target and nums[right]>target:
                left=mid+1
            else:
                right=mid
        return -1